File: C#/백준/Bronze/4562. No Brainer/No Brainer.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            for(int i =0; i<a; i++){
                string[] aa = Console.ReadLine().Split();
                int b = int.Parse(aa[0]);
                int c = int.Parse(aa[1]);
                if(c>b){Console.WriteLine("NO BRAINS");}
                else{Console.WriteLine("MMM BRAINS");}
            }
        }
    }
}

File: Java/백준/Bronze/4562. No Brainer/No Brainer.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        for(int i =0; i<a; i++){
            int b = in.nextInt(); int c = in.nextInt();
            if(c>b){
                System.out.println("NO BRAINS");
            }
            else{
                System.out.println("MMM BRAINS");
            }
        }
    }
}